e2e_test/streaming/project_set.slt

```diff
@@ -107,3 +107,27 @@ with cte as (SELECT 1 as v1, unnest(array[1,2,3,4,5]) AS v2) select v1 from cte;
 1
 1
 1
+
+statement ok
+create table t(arr varchar[]);
+
+statement ok
+create materialized view mv as select * from t cross join unnest(t.arr) WITH ORDINALITY AS x(elts, num);
+
+statement ok
+insert into t values (Array['a','b', 'c']), (Array['d','e']);
```

Comment:
What if update `Array['a','b', 'c']` to `Array['a', 'c']`?

Reply:
Updated. Actually I'm not sure what's your intention 🤔